### DISCUSSION TITLE
Allow MySQL/MariaDB as Database back-end

### DISCUSSION THREAD

**Discussion by loewexy** [jonaswinkler/paperless-ng#1175] [Feature Requests]
It would be nice to have support for MySQL/MariaDB. Since most OpenSource Software uses that, I already have a DB server with backups and stuff configured in my network. Since you already support Postgres and SQLite I guess that should not be to difficult.

**Comment by suhrmann** [comment]
> Django [3.2] officially supports the following databases:
> - PostgreSQL
> - MariaDB
> - MySQL
> - Oracle
> - SQLite

https://docs.djangoproject.com/en/3.2/ref/databases/ 

The `main` of Paperless-ng currently uses [django==3.2.4](https://github.com/jonaswinkler/paperless-ng/blob/master/requirements.txt#L35), so the database middleware should actually™ support MariaDB as well...

Does the Paperless-ng with MariaDB throw errors?

**Reply by Conspir3D** [reply]
has someone successfully managed to run paperless-ng on mysql/mariadb ?
